web/src/App.tsx

```diff
@@ -15,21 +21,50 @@ const App: React.FunctionComponent = () => (
                     <Link className="metamist-img" to="/">
                         <MuckTheDuck height={28} style={{ marginRight: '5px' }} />
                     </Link>
+
                     <Link className="metamist" to="/">
```

Comment:
Should this be refactored into a Header component as it's now getting bigger and more complicated.

Reply:
Added NavBar component